app/models/application_search.rb

```diff
@@ -39,4 +37,10 @@ def datastore_params
       sorting: @sorting.to_h
     }
   end
+
+  def datastore_response
+    http_client.post('/searches', datastore_params)
```

Comment:
Not using API client for the searches? Happy to adapt it so you can use it if needed.

Reply:
Thank you @zheileman! We're still in a bit of flux. I'll let you know as soon as we're a bit more certain about requirements!

Reply:
No prob!